packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -1066,7 +1070,14 @@ export class GraphiQL extends React.Component<GraphiQLProps, GraphiQLState> {
         );
       }
 
-      const totalResponse: FetcherResult = { data: {} };
+      const isIncrementalDelivery = (
+        result: FetcherResult,
+      ): result is IncrementalDeliveryResult => {
+        // @ts-ignore
+        return typeof result === 'object' && 'hasNext' in result;
```

Comment:
`typeof n === 'object'` is super risky. for example: `const n = null; typeof n === 'object' // true`